doc/pfc_asym/PFC_Asymmetric_Test_HLD.md

```diff
@@ -428,3 +429,27 @@ setup, pfc_storm_runner, enable_pfc_asym
 
 - Teardown:
   - Stop ARP responder
+
+### Test case # 5 - Priority Group dropped packets testing
+#### Test objective
+Asymmetric PFC is disabled. Verify ingress Priority Group drop packets counter increasing
+
+#### Used fixtures
+setup, pfc_storm_runner
+
+#### Test steps
+- Setup:
+  - Start ARP responder
+  - Limit maximum bandwith rate on the destination port by setting "1" into SAI port attribute SAI_PORT_ATTR_QOS_SCHEDULER_PROFILE_ID
```

Comment:
Please clarify the configuration needed, SAI_PORT_ATTR_QOS_SCHEDULER_PROFILE_ID needs a value of type sai_object_id_t (https://github.com/opencomputeproject/SAI/blob/master/inc/saiport.h#L1366). What is "1" in this case?
Please mention that this test requires RPC sai thrift to perform this configuration.